src/pages/createStore.ts

```diff
@@ -50,7 +50,10 @@ function loadStore(loadedStore: SavedState | undefined) {
         octokit: loadedStore.session.githubAccessToken
           ? generateOctokitInstance(loadedStore.session.githubAccessToken)
           : undefined
-      }
+      },
+      googleUser: loadedStore.session.googleAccessToken
+        ? 'placeholder' // updates in PersistenceSaga
```

Comment:
This should not be a placeholder; either we set it to a value, or don't set it at all (i.e. use the defaults).